multipath-tools/src/go/start_multipath.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "syscall"
+)
+
+func main() {
+    fmt.Println("Starting multipath daemon...")
+
+    //setup a chroot environment for the extension
```

Comment:
i don't think we need this, with `mountPropogation` shared should work as a normal extension service